functions/src/world.ts

```diff
@@ -172,13 +172,16 @@ export const addWorldAdmin = functions.https.onCall(async (data, context) => {
 
   const owners = worldData.owners as string[];
 
-  if (!Array.isArray(owners) || owners.includes(userId)) {
+  if (!Array.isArray(owners)) {
     return;
   }
 
-  const newOwners = [...owners, userId];
+  const newOwners = [...owners, ...userIds];
+
+  //Removes duplicates from the concat array.
+  const uniqueOwners = [...new Set(newOwners)];
```

Comment:
[NOTE]
No need to change this code, just pointing out if we use Lodash for functions, seeing function named `uniq` would replace the need of an explanation comment. This also means if we see too many of these [...new Set()] snippets, we might make our own `unique` function.